### PR TITLE
feat: Add vehicle endpoint

### PR DESCRIPTION
**Pull request by kk5190**
Implements vehicles endpoint feature #151

### CODE REVIEW

File: modules/vehicles/api/vehicles-routes.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import { getQtyFromRequest } from '../../../utils/route-utils';
+import getRandomVehicles from '../utils/getRandomVehicles';
+
+module.exports = function (app: core.Express) {
+  //Get a randomly generated user
+  app.get('/vehicles/:qty?', (req: Request, res: Response) => {
```
Comment (ageddesi) [inline modules/vehicles/api/vehicles-routes.ts:8]: We no longer are manually updating swagger.json as such please can you add OpenAPI comments for the api endpoint here and also in your vehicle.ts file.

Comment (ageddesi) [reply modules/vehicles/api/vehicles-routes.ts:8]: Secondly can you add tests for the end point please, examples of which exist in other modules. 
I have updated the target branch of this merge to v0,13.0 which is where the latest code lives. 👍 

Thanks for the hard work

**Review by ageddesi — CHANGES_REQUESTED** [review]

File: modules/vehicles/api/vehicles-routes.ts (hunk 1 above)
Comment (kk5190) [reply modules/vehicles/api/vehicles-routes.ts:26]: Thanks, this PR is now updated @ageddesi

**Comment by ageddesi** [comment]
Thank you for the amazing work on this @kk5190 Happy hacktoberfest 👍

**Pull request merged**